sites/public/__tests__/components/listing/ListingViewSeeds.test.tsx

```diff
@@ -14,4 +15,33 @@ describe("<ListingViewSeeds>", () => {
     const view = render(<ListingViewSeeds listing={listing} jurisdiction={jurisdiction} />)
     expect(view.getByRole("heading", { level: 1 })).toHaveTextContent(listing.name)
   })
+
+  it("doesn't show listing updated at for a default jurisdiction", () => {
+    const view = render(<ListingViewSeeds listing={listing} jurisdiction={jurisdiction} />)
+    expect(view.queryByText("Listing Updated: Dec 31, 2019")).toBeNull()
+  })
+
+  it("shows listing updated at when the jurisdiction allows it", () => {
+    const view = render(
+      <ListingViewSeeds
+        listing={listing}
+        jurisdiction={{
+          ...jurisdiction,
+          featureFlags: [
+            ...jurisdiction.featureFlags,
+            {
+              id: "test_id",
+              createdAt: new Date(),
+              updatedAt: new Date(),
+              description: "",
+              active: true,
+              jurisdictions: [],
+              name: FeatureFlagEnum.enableListingUpdatedAt,
+            },
+          ],
+        }}
+      />
+    )
+    expect(view.getAllByText("Listing Updated: Dec 31, 2019")).toBeDefined()
```

Comment:
nit: If there is more than one I like to have an explicit expect for the length. So this would become
```
    expect(view.getByText("Listing Updated: Dec 31, 2019")).toHaveLength(2)
```

Also, I do find it weird that there is a second "Listing Updated" in the DOM. I see that when I render the page but the other instance is not visible by the user. Do you know why that is?

Reply:
@ludtkemorgan hmm, somehow I dismissed that as being weird because I assumed there was some duplicated DOM that was shown/hidden depending on viewport size, but I wonder if that's actually the case. I'll take a look at it!

Reply:
Yeah, that was the reason. I've updated the test to check specifically for the length 2.